cmd/simulator/load/sequences.go

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils"
+	"github.com/ava-labs/avalanchego/utils/math"
+	"github.com/ava-labs/avalanchego/vms/platformvm"
+	pwarp "github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	"github.com/ava-labs/subnet-evm/cmd/simulator/config"
+	"github.com/ava-labs/subnet-evm/cmd/simulator/txs"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ethereum/go-ethereum/log"
+	"golang.org/x/exp/maps"
+)
+
+func GetWarpSendTxSequences(
+	ctx context.Context, config config.Config, chainID *big.Int,
+	pks []*ecdsa.PrivateKey, startingNonces []uint64,
```

Comment:
Can we update the function signatures so that they are either all on one line or each param is on its own line?